desktop-environment.el

```diff
@@ -176,6 +193,17 @@ portion of the screen."
   "Shell command toggling bluetooth."
   :type 'string)
 
+
+;;; Helper variables - keyboard layout
+
+(defvar desktop-environment-keyboard-layouts-ring []
```

Comment:
please make the variable private by using 2 dashes: `desktop-environment--keyboard-layouts-ring`